microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -985,7 +985,8 @@ def verify_no_pre_exist_users(self, node: Node) -> None:
             passwd_matches = re.split(":", passwd_raw_output)
             file_path = f"{passwd_matches[5]}/.ssh/authorized_keys"
             file_exists = node.tools[Ls].path_exists(file_path, sudo=True)
-            if file_exists:
+            stat = node.tools[Stat]
+            if file_exists and stat.get_total_size(file_path, sudo=True) > 0:
```

Comment:
we can move stat into if block to reduce the time, just use this tool if file exists

Reply:
I changed to check if the key_content is empty.